Firestore/core/test/unit/FSTGoogleTestTests.mm

```diff
@@ -82,10 +82,27 @@ @interface GoogleTests : XCTestCase
     return nil;
   }
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000 || \
+    __MAC_OS_X_VERSION_MAX_ALLOWED >= 101300
```

Comment:
Since we no longer support iOS 8, could this can be phrased as an `@available` test?

Also, this needs a case for tvOS 11.

Reply:
Thank you, I added it and it works locally.